src/components/JoinUs/Perks.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+import styled from 'styled-components'
+import remcalc from 'remcalc'
+import breakpoint from 'styled-components-breakpoint'
+import { Row, Col } from '../grid'
+import { Padding } from 'styled-components-spacing'
+import { Paragraph, H2 } from '../Typography'
+import Image from '../Common/Image'
+
+import { Section, ShiftedColumn } from './elements'
+
+const PerksListContainer = styled(ShiftedColumn)`
+  display: grid;
+  row-gap: ${remcalc(24)};
+  grid-template-columns: 1fr 1fr;
+  column-gap: 42px;
+  padding-top: ${remcalc(24)};
+
+  ${breakpoint('smallPhone')`
+    grid-template-columns: 1fr 1fr;
+  `};
+
+  ${breakpoint('smallTablet')`
+    grid-template-columns: 1fr 1fr 1fr;
+  `};
+
+  ${breakpoint('tablet')`
+    grid-template-columns: 1fr 1fr;
+    row-gap: ${remcalc(36)};
+    column-gap: 42px;
+    padding-top: ${remcalc(12)};
+  `};
+`
+
+const Icon = styled(Image)`
+  width: 60px;
+  height: 60px;
```

Comment:
I think I'm right in saying that we currently have multiple ways to achieve this restricted size across our codebase right now.

Perhaps it's worth us add pattern, of specifying/restricting the width & height, via a prop on Image. What do you think?

Could also be the remit of a separate ticket. Just throwing it out there.

Reply:
@jsms90 sure I think it makes sense, should cover most of the solutions.